third_party/pulumi/doc.go

```diff
@@ -0,0 +1,3 @@
+// The License of Pulumi is Apache License 2.0 (https://github.com/pulumi/pulumi/blob/master/LICENSE)
```

Comment:
the file is useless, because #190 remove 3rd-party and restore to go mod dependency

Reply:
ok, fixed